sourcetools/com.ibm.uma/com/ibm/j9/uma/Main.java

```diff
@@ -224,9 +224,9 @@ public static void main(String[] args) {
 		dumpSplash();
 		
 		if ( jitVersionFile.equals("") ) {
-			File tempJitVersionFile = new File(rootDir + "/tr.source/codegen/version.h");
+			File tempJitVersionFile = new File(rootDir + "/compiler/codegen/version.h");
 			if(tempJitVersionFile.exists() && !tempJitVersionFile.isDirectory()) {
-				jitVersionFile = rootDir + "/tr.source/codegen/version.h";
+				jitVersionFile = rootDir + "/compiler/codegen/version.h";
```

Comment:
Neither of these paths exist any more.  If it still needs to run perhaps the initial path should be `/compiler/trj9/build/version.h`.

Reply:
Fixed in 626683a